src/logic/safe/store/actions/processTransaction.ts

```diff
@@ -134,11 +136,12 @@ export const processTransaction =
 
       transaction = isExecution ? getExecutionTransaction(txArgs) : getApprovalTransaction(safeInstance, tx.safeTxHash)
 
+      const gasParam = getNetworkId() === ETHEREUM_NETWORK.MAINNET ? 'maxFeePerGas' : 'gasPrice'
```

Comment:
We really need to refactor `createTransaction` and `processTransaction`. 95% of the code is the same.